src/matter/cluster/server/OnOffServer.ts

```diff
@@ -0,0 +1,160 @@
+/**
+ * @license
+ * Copyright 2022 The node-matter Authors
+ * SPDX-License-Identifier: Apache-2.0
+ */
+
+import {
+    OnOffCluster, OnOffDelayedAllOffEffectVariant, OnOffDyingLightEffectVariant, OnOffEffectIdentifier
+} from "../OnOffCluster";
+import { ClusterServerHandlers } from "./ClusterServer";
+import { AttributeServer } from "./AttributeServer";
+
+/*
+TODO: Global Cluster fields needs to be added also here because, as discussed, based on the implementation.
+* Cluster Revision: 4 (If I get it right from the Specs - Application Cluster Specs 1.5.1)
+* FeatureMap: Bit 0 should be set when it is a "lighting" usecase AND Level Control cluster is supported
+* AttributeList:
+  * Supported always: onOff, onTime, offWaitTime
+  * Supported if we have a scene cluster: globalSceneControl
+  * Not supported: startUpOnOff
+* AcceptedCommandList:
+  * Supported always: Off, On, Toggle, OnWithTimedOff
+  * Supported if we have a scene cluster: onWithRecallGlobalScene
+  * Supported if we have a Level cluster: OffWithEffect
+* GeneratedCommandList: empty
+* EventList: empty
+* FabricIndex: empty
+ */
+
+/*
+TODO: If the Scenes server cluster is implemented on the same endpoint, the following extension field SHALL
+      be added to the Scene Table:
+      * OnOff
+ */
+
+export const OnOffClusterHandler: () => ClusterServerHandlers<typeof OnOffCluster> = () => ({
+    // TODO In fact for the Cluster server implementation we need "initialization code" or thinks like this.
```

Comment:
I need somehow an "instance" representation of the handlers ... so in fact would be the ClusterServer instance or such

Reply:
cool ... exactly this was the "missing puzzle piece"

Reply:
lets talk in our call ... I miss how I can access the attributes from the cluster in the handler "constructor"